pkg/arvo/lib/ph/azimuth.hoon

```diff
@@ -183,31 +248,68 @@
     --
   --
 ::
-++  dawn
+++  raw-real-ship
   |=  who=ship
+  =/  m  (ph ,~)
+  ^-  form:m
+  ?.  =(%earl (clan:title who))
+    (raw-ship:(ph-tests our) who `(dawn who ~))
+  =/  spon=ship  (^sein:title who)
+  =/  cub  (pit:nu:crub:crypto 128 (sham who life=1 %entropy))
```

Comment:
Also 128-bits here, although I'm not sure that matters.

Reply:
it's just a test, but may as well.